bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -285,7 +285,11 @@ spec:
                   if [[ -f "${MARIADB_ROOT_PASSWORD_FILE:-}" ]]; then
                       password_aux=$(cat "$MARIADB_ROOT_PASSWORD_FILE")
                   fi
-                  exec mysqladmin status -u"${MARIADB_ROOT_USER}" -p"${password_aux}"
+                  # If wsrep_local_state != 4, would mean the node is not yet synced and therefore not ready to receive connections
+                  if [[ ! $(mysqladmin extended-status -u"${MARIADB_ROOT_USER}" -p"${password_aux}" | grep " wsrep_local_state " | grep " 4 ") ]]; then
```

Comment:
I would remove the white spaces around the grep strings.
Also maybe you want to change it and use `wsrep_local_state_comment` instead.
```
| wsrep_local_state                                      | 4                                                                                                                                              
| wsrep_local_state_comment                              | Synced            
```